sdk/communication/AzureCommunicationChat/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0-beta.12 (2021-05-03)
+### New Features
+- Change generated codes to internal access level.
```

Comment:
This isn't a feature, it's a whole bunch of breaking changes that need to be spelled out. Which models are no longer visible? The change in how customers would instantiate an options object (i.e. FooOptions instead of Foo.FooOptions), etc.